modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/causalitydatanodes/DistributionZoneCausalityDataNodesTest.java

```diff
@@ -1205,7 +1207,7 @@ private void checkDataNodesRepeated(
 
         // Above, we have waited for data nodes in meta storage to become the same as in expectedDataNodesAfterTimersAreExpired map,
         // so we use the applied revision and check the value of data nodes in data nodes manager.
-        HybridTimestamp timestamp = metaStorageManager.timestampByRevisionLocally(metaStorageManager.appliedRevision());
+        HybridTimestamp timestamp = metaStorageManager.timestampByRevisionLocally(keyValueStorage.revision());
```

Comment:
please review all comments were `topology augmentation` is mentioned

Reply:
fixed